src/pubtools/_marketplacesvm/cloud_providers/ms_azure.py

```diff
@@ -338,6 +338,20 @@ def _post_publish(
 
         return push_item, update_res
 
+    def _delete(self, push_item, **kwargs):
+        """
+        Not implemented for Azure. Passes back push_item without changing anything.
+
+        Args:
+            push_item (VHDPushItem)
+                The original push item for uploading the image.
+            builds (List[str])
+                The builds to delete.
+        Returns:
+            A VHDPushItem
+        """
+        return push_item
```

Comment:
What about adding a `#TODO` comment here? Just for the sake of completion

Or returning `NotImplementedError`? Then it won't successfully complete the task giving the false impression that the VHD was deleted

Reply:
Added todo and a log.info explaining it.

Reply:
Thanks!